xbmc/cores/VideoPlayer/VideoPlayer.cpp

```diff
@@ -2475,14 +2475,17 @@ void CVideoPlayer::HandleMessages()
     {
       CDVDMsgOpenFile &msg(*static_cast<CDVDMsgOpenFile*>(pMsg));
 
+      CJobManager::GetInstance().Submit([this]() {
+        m_callback.OnPlayBackStarted();
```

Comment:
Was the removed call to `m_callback.OnPlayBackStopped()` a few lines down wrong or should this have been `m_callback.OnPlayBackStopped()` instead of `m_callback.OnPlayBackStarted()`?

Reply:
The call a few lines down was introduced yesterday and was wrong. Calling the started callback instead if stopped seems to be correct. Also moving this dime lines up seems okay to me. 

Additionally, I runtime tested the changes and could not find any suspicious change in behavior.

Reply:
the change here is correct. playback is not stopped if we call OpenFile while playing.